testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -17,6 +17,7 @@
 from six.moves import zip
 
 from collections import namedtuple
+import pickle
```

Comment:
Shouldn't this be using pickle from `six`?

Reply:
good catch. Haven't thought about it.